.github/workflows/release.yml

```diff
@@ -0,0 +1,217 @@
+# Copyright (c) 2010-2022, Lawrence Livermore National Security, LLC. Produced
+# at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+# LICENSE and NOTICE for details. LLNL-CODE-443271.
+#
+# This file is part of the GLVis visualization tool and library. For more
+# information and source code availability see https://glvis.org.
+#
+# GLVis is free software; you can redistribute it and/or modify it under the
+# terms of the BSD-3 license. We welcome feedback and contributions, see file
+# CONTRIBUTING.md for details.
+
+# GLVis - an OpenGL visualization server based on the MFEM library
+
+name: release
+
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+
+env:
+  HYPRE_ARCHIVE: v2.19.0.tar.gz
+  HYPRE_TOP_DIR: hypre-2.19.0
+  METIS_ARCHIVE: metis-4.0.3.tar.gz
+  METIS_TOP_DIR: metis-4.0.3
+  MFEM_TOP_DIR: mfem
+  MFEM_BRANCH: master
+
+jobs:
+  release-builds:
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-10.15, windows-2022]
```

Comment:
I see Ubuntu here, but no binary for it... what am I missing?

Reply:
Is this the issue? (from https://github.com/GLVis/glvis/actions/runs/2348149119)

<img width="952" alt="Screen Shot 2022-05-18 at 3 06 36 PM" src="https://user-images.githubusercontent.com/1278247/169163712-eadf7e2f-123b-466c-b31d-db71c8a9c76b.png">

Reply:
I was planning on looking into an Appimage build with Ubuntu, but right now the ubuntu action doesn't do anything. I'll just remove it for now.

Reply:
Okay, got it.

(I also hear good things about `snap`)